Tests/GitHubContents.tests.ps1

```diff
@@ -172,6 +179,176 @@ try
 
         Remove-GitHubRepository -Uri $repo.svn_url -Confirm:$false
     }
+
+    Describe 'GitHubContents/Set-GitHubContent' {
```

Comment:
I **really** appreciate the awesome and thorough UT's that you've been writing for all of your submissions.  SO, so much appreciated.  Thank you.